### DISCUSSION TITLE
Is there a way to join on a subquery

### DISCUSSION THREAD

**Discussion by Xstoudi** [adonisjs/core#3945] [Help]
For example:
```
SELECT *
FROM ...
INNER JOIN (
     SELECT ...
    FROM ...
    LIMIT ...
) AS ... ON ... = ...
```

**Comment by joeyjurjens** [comment]
I think you mean this: https://docs.adonisjs.com/reference/database/query-builder#join ?

**Reply by Xstoudi** [reply]
No, I'm not trying to join on a table that already exists, but on the returned rows of a sub-query.
